core/src/main/java/com/radixdlt/ledger/StateComputerLedger.java

```diff
@@ -371,22 +352,16 @@ private void commit(
         return;
       }
 
-      var verifiedExtension =
-          verifier.verifyAndGetExtension(
-              againstLedgerHeader.getAccumulatorState(),
-              committedTransactionsWithProof.getTransactions(),
-              t -> t.getLegacyPayloadHash().inner(),
-              committedTransactionsWithProof.getProof().getAccumulatorState());
-
-      if (verifiedExtension.isEmpty()) {
-        throw new ByzantineQuorumException(
-            "Accumulator failure " + againstLedgerHeader + " " + committedTransactionsWithProof);
-      }
+      final var transactions = committedTransactionsWithProof.getTransactions();
+      final var startIndex =
+          transactions.size()
+              - nextHeader.getAccumulatorState().getStateVersion()
+              + againstLedgerHeader.getAccumulatorState().getStateVersion();
+      final var extension = transactions.subList(Ints.checkedCast(startIndex), transactions.size());
 
-      var transactions = verifiedExtension.get();
       var extensionToCommit =
```

Comment:
No, not possible.

Line 351 (after PR) here does basically `if (nextHeader.stateVersion <= againstLedgerHeader.stateVersion) return;`.
In the above, if we do `X = nextHeader.stateVersion - againstLedgerHeader.stateVersion` then we have `X > 0`.
Which means that `startIndex = transactions.size() - X` must be strictly smaller than `transactions.size()`.
Which also kinda guarantees that `transactions` is not empty (because if it was, then `.subList(X, 0)` would throw an IOOBE.